### ISSUE TITLE
Landing

### ISSUE THREAD

**Issue by zubairabid** [zubairabid/megathon2018#2]
Not too sure about the animation. Also, implement fade in/fade out to complement the parallax

**Comment by zubairabid** [comment]
review performance of fade in.

**Comment by abhigyanghosh30** [comment]
![screenshot from 2018-06-15 00-24-59](https://user-images.githubusercontent.com/30973042/41432166-d44bcc40-7032-11e8-9d53-d3744e47c961.png)
Landing seems to break down in some cases

**Comment by zubairabid** [comment]
@abhigyanghosh30 That's an issue with object-fit and the default box-shadow for nav elements

**Comment by abhigyanghosh30** [comment]
@zubairabid also the image is not centered for smaller devices. Can we do that as well?

**Comment by zubairabid** [comment]
@abhigyanghosh30 Yes, that's the issue with object-fit. We aren't using background-image for reasons  (primarily materialize css being a bitch about standard methods of opaque overlays) and so the next solution is to use a div container with the image as object-fit, which we're doing. Only, it doesn't work for some reason. 

A simple and insufficient fix would be to make the container a flex and justify, align as center. Then set image height as 100%. However, if the width of the screen is greater than the width of the image then there is empty space left over. Note that this method can be done with more than just flexbox, if you don't like it (use flexbox tho it's rampamtly abused on whatever we're doing)

We can  implement the latter fix temporarily until we figure out object-fit's issue.

**Comment by zubairabid** [comment]
@abhigyanghosh30 should be fixed
